### PR TITLE
Update response for database api

### PR DESCRIPTION
**Pull request by imnutz**
Ad `user_id` and `description` to the response

### CODE REVIEW

**Review by tung-vu-td — APPROVED** [review]
LGTM

**Review by tung-vu-td — APPROVED** [review]
LGTM

File: src/main/java/com/treasuredata/client/model/TDDatabase.java [hunk 1]
```diff
@@ -30,6 +30,8 @@ public class TDDatabase
     private final String id;
     private final String name;
     private final long count;
+    private final long userId;
```
Comment (yuokada) [inline src/main/java/com/treasuredata/client/model/TDDatabase.java:33]: In td-client-java, `user_id` is handled as an integer value as follows:
https://github.com/treasure-data/td-client-java/blob/75366cdfe7eba1018aeb4d253ba045fb905cebc8/src/main/java/com/treasuredata/client/model/TDApiKey.java#L16

It's good to follow the existing data type for `user_id` as much as possible.

**Review by yuokada — APPROVED** [review]

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

**Pull request merged**